### PR TITLE
Changes which service logs Filebeat collects, and add tags

### PR DESCRIPTION
**Pull request by Ruoyu111**
Modified the Filebeat configuration in the openstack controller node. Let Filebeat collect different log files from openstack services and also add tags on them.

### CODE REVIEW

File: quickstack/manifests/controller_common.pp [hunk 1]
```diff
@@ -907,8 +909,31 @@
...
+    }
+
+   filebeat::prospector {'openstack-log':
+      paths => [
+        '/var/log/nova/*.log',
+        '/var/log/glance/*.log',
+        '/var/log/cinder/*.log',
+	'/var/log/keystone/*.log',
+	'/var/log/neutron/*.log',
+      ],
```
Comment (rahulait) [inline quickstack/manifests/controller_common.pp:927]: proper spacing needed on line 926 and 927 here.

File: quickstack/manifests/controller_common.pp [hunk 2]
```diff
@@ -907,8 +909,31 @@
     }
   }
 
-   filebeat::prospector { 'generic':
-      paths => ["/var/log/*.log", "/var/log/secure", "/var/log/messages", "/var/log/ceph/*", "/var/log/nova/*", "/var/log/neutron/*", "/var/log/openvswitch/*", "/var/log/cinder/*", "/var/log/glance/*", "/var/log/horizon/*", "/var/log/httpd/*", "/var/log/keystone/*"]
+   filebeat::prospector {'syslog':
+      paths => [
+	'/var/log/*.log',
+	'/var/log/secure',
+	'/var/log/messages',
```
Comment (rahulait) [inline quickstack/manifests/controller_common.pp:915]: 2 spaces on line 914, 915 and 916 instead of 1

**Comment by rahulait** [comment]
You can close this pull request and open a new pull request with 1 commit.

**Comment by Ruoyu111** [comment]
Okay, I will do it now. Thanks!

**Pull request closed without merging**